components/leds.py

```diff
@@ -0,0 +1,401 @@
+import math
+import wpilib
+import time
+from enum import Enum, auto
+from utilities.scalers import scale_value
+import random
+
+
+MAX_BRIGHTNESS = 100  # Between 0-255 of Value on HSV scale
+
+
+class LedColors(Enum):
+    # Use HSV to get nicer fading, hues are 0-180 so half usual hue
+    RED = (0, 255, MAX_BRIGHTNESS)
+    ORANGE = (20, 255, MAX_BRIGHTNESS)
+    YELLOW = (30, 255, MAX_BRIGHTNESS)
+    PINK = (150, 255, MAX_BRIGHTNESS)
+    BLUE = (120, 255, MAX_BRIGHTNESS)
+    CYAN = (90, 255, MAX_BRIGHTNESS)
+    GREEN = (60, 255, MAX_BRIGHTNESS)
+    WHITE = (0, 0, MAX_BRIGHTNESS)
+    VIOLET = (300, 45, MAX_BRIGHTNESS)
```

Comment:
Is this hue correct?

Reply:
[Yep](https://colorpicker.me/#ff8cff)

Reply:
[Are you sure?](https://github.com/thedropbears/pychargedup/pull/22/files#diff-a5d4d89c877b5b89f11a6badf91fc53817c427e53edc09b301978a01893d1764R13)